web/views.py

```
@@ -7599,3 +7601,56 @@ def add_contributor(username, avatar_url, profile_url):
         print(f"Error fetching contributors: {e}")
         # Return an empty list in case of error
         return render(request, "web/contributors_list.html", {"contributors": []})
+
+
+def video_request_list(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `video_request_list`

(ANN201)